rosa_getting_started/rosa-deleting-cluster.adoc

```diff
@@ -7,4 +7,13 @@ toc::[]
 
 Delete a {product-title} (ROSA) cluster using the `rosa` command-line.
 
+[id="prerequisites_rosa-deleting-cluster"]
+== Prerequisites
+
+* If {product-title} created a VPC, you must remove the following before you delete your cluster:
+  * network configurations, such as VPN configurations and VPC peering
```

Comment:
Should "VPC peering" be something like "VPC peering connections" or "VPC peering configurations"?

Reply:
@0kashi - which one should it be?

Reply:
VPC peering connections is fine